translations/en.json

```
@@ -356,7 +356,7 @@
   "rule.desc.NoHeadings": "<p>If appropriate, add headings to the page to organize the content for sighted and <a href='https://en.wikipedia.org/wiki/Screen_reader' target='_blank'>screen reader</a> users. The headings should represent an accurate outline of the content</p>",
   "rule.desc.ObjectMustContainText": "<p>Multimedia objects should be accompanied by a link to a transcript of the content.</p>",
   "rule.desc.ObjectTagDetected": "<p>Multimedia embedded using the 'Object' tag may require the user to install a plugin for their web browser. This can create support and access issues for some users. Additionally, users on mobile devices may not be able to view the multimedia content at all. Consider using an alternative format that the user's browser can display natively.</p>",
-  "rule.desc.ParagraphNotUsedAsHeader": "<p>Bold and Italics are used to emphasize text, whereas headings are used to define the structure of the document. Headings like <code>h1-h6</code> are extremely useful for non-sighted users to navigate the structure of the page, and formatting a paragraph to just be big or bold, while it might visually look like a heading, does not make it one.</p>",
+  "rule.desc.ParagraphNotUsedAsHeader": "<p>Bold and Italics are used to emphasize text, whereas headings are used to define the structure of the document. Headings like <code>h1-h6</code> are extremely useful for non-sighted users to navigate the structure of the page, and formatting a paragraph to just be big or bold, while it might visually look like a heading, does not make it one. UDOIT has flagged this as a potential usage of style for document structure, please ensure that it is not the case.</p>",
```

Little nit-picky and might be overkill, but I feel this'll make it clearer to the user what they should look for in their html. 

```suggestion
  "rule.desc.ParagraphNotUsedAsHeader": "<p>Bold and Italics are used to emphasize text, whereas headings are used to define the structure of the document. Headings like <code>h1-h6</code> are extremely useful for non-sighted users to navigate the structure of the page, and formatting a paragraph to just be big or bold, while it might visually look like a heading, does not make it one. UDOIT flagged this as a potential usage of style for document structure, please ensure that this is not the case when you review the html editor in the page.</p>",
```

---

I'll make that change!